src/js/actions/AppsActions.js

```diff
@@ -135,11 +135,12 @@ var AppsActions = {
   },
   applySettingsOnApp: function (appId, settings, isEditing = false) {
     // Version key is not allowed and not needed on settings object
-    delete settings.version;
+    var clonedSettings = Object.assign({}, settings);
```

Comment:
Yeah the function name makes it a bit confusing, but if you pass just one object you're basically cloning it.